src/Sfa.Das.ApprenticeshipInfoService.Api/App_Start/SwaggerConfig.cs

```diff
@@ -118,7 +119,7 @@ public static void Register()
                         // If you want to post-modify "complex" Schemas once they've been generated, across the board or for a
                         // specific type, you can wire up one or more Schema filters.
                         //
-                        //c.SchemaFilter<ApplySchemaVendorExtensions>();
+                        c.SchemaFilter<ApplySchemaVendorExtensions>();
```

Comment:
So in Swagger UI it ignores properties with an obsolete attribute but the API still delivers them